packages/fxa-shared/scripts/feature-flags.js

```diff
@@ -10,6 +10,7 @@ const Ajv = require('ajv');
 const ajv = new Ajv();
 const Redis = require('ioredis');
 const redis = new Redis({
+  password: process.env.REDIS_PASSWORD || 'fxa123',
   host: process.env.REDIS_HOST || 'localhost',
   port: process.env.REDIS_PORT || 6379,
```

Comment:
~Possibly~ Guaranteed dumb question, but why reading from `process.env.***` here versus the convict config above?
Although maybe concerns since one is .js and other is .ts. Plus, I guess it's existing code, so... not relevant to this PR.

Reply:
Yes, I had the same question but think it's out of scope. I am guessing that since this is just a 'script' pulling in the entire convict config might be overkill?